### PR TITLE
Twitter is now X

### PR DESCRIPTION
**Pull request by komi1230**
Sadly, there is no Twitter in this world anymore.

### CODE REVIEW

**Comment by zkochan** [comment]
We can change the title but why should we change the domains if they just redirect to `twitter.com`?

**Comment by komi1230** [comment]
@zkochan 
Thank you for checking. 
I think `x.com` will be the main domain for X Corp.

File: pnpm/CHANGELOG.md
Comment (zkochan) [inline pnpm/CHANGELOG.md:1]: Don't modify the changelogs.

**Review by zkochan — CHANGES_REQUESTED** [review]

**Comment by ExE-Boss** [comment]
Everyone keeps calling it **Twitter**, so I’m not particularly keen on this change.

**Pull request closed without merging**